archive/html-archive/istqb-ctel-tm-sm-complete-guide.html

```
@@ -0,0 +1,2149 @@
+
+  <rect x="410" y="20" width="170" height="160" rx="8" fill="rgba(255,204,0,0.06)" stroke="rgba(255,204,0,0.3)" stroke-width="1.5"/>
+  <text x="495" y="42" text-anchor="middle" fill="#ffcc00" font-size="10" font-weight="700" font-family="'Space Mono', monospace">内部失敗コスト</text>
+  <text x="495" y="58" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">30-40%が理想</text>
+  <text x="495" y="74" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">バグ修正・再テスト</text>
+  <text x="495" y="89" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">リリース遅延コスト</text>
+
+  <rect x="600" y="20" width="180" height="160" rx="8" fill="rgba(255,69,58,0.08)" stroke="rgba(255,69,58,0.4)" stroke-width="1.5"/>
+  <text x="690" y="42" text-anchor="middle" fill="#ff453a" font-size="10" font-weight="700" font-family="'Space Mono', monospace">外部失敗コスト 🚨</text>
+  <text x="690" y="58" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">10-20%（低いほどよい）</text>
+  <text x="690" y="74" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">本番障害・サポート</text>
+  <text x="690" y="89" text-anchor="middle" fill="#8b949e" font-size="9" font-family="'IBM Plex Sans JP', sans-serif">賠償・ペナルティ</td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </td> ]

(tag-pair)